formik-mui/package.json

```diff
@@ -1,94 +1,99 @@
-  "directories": {
-    "test": "tests"
-  },
-  "scripts": {
-    "build:commonjs": "cross-env BABEL_ENV=commonjs babel src --out-dir lib --presets=@babel/preset-env,@babel/preset-react",
-    "build:es": "babel src --out-dir es",
-    "build:umd": "cross-env NODE_ENV=development rollup -c -o dist/formik-material-fields.js",
-    "build:umd:min": "cross-env NODE_ENV=production rollup -c -o dist/formik-material-fields.min.js",
-    "build": "npm run build:commonjs && npm run build:es",
-    "clean": "rimraf lib dist es coverage",
-    "prepare": "npm run clean && npm run build",
-    "dev": "rollup -c -w",
-    "test": "node tests/test.js",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook",
-    "deploy-storybook": "cd .. && storybook-to-ghpages --packages . && cd -"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/settlin/node-monorepo/formik-mui.git"
-  },
-  "keywords": [
-    "formik",
-    "material",
-    "form",
-    "field"
-  ],
-  "author": {
-    "name": "Settlin",
-    "email": "developers@settlin.in"
-  },
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/settlin/node-monorepo/formik-mui/issues"
-  },
-  "homepage": "https://github.com/settlin/node-monorepo/formik-mui#readme",
-  "peerDependencies": {
-    "@material-ui/core": ">=4",
-    "@material-ui/icons": ">=4",
-    "@material-ui/lab": ">=4.0.0-alpha.58",
-    "formik": ">=1",
-    "react": ">=16.8",
-    "react-dom": ">=16.8"
-  },
-  "dependencies": {
-    "@date-io/dayjs": "^1.3.13",
-    "@tinymce/tinymce-react": "^3.6.1",
-    "clsx": "^1.1.1",
-    "dayjs": "^1.8.36"
-  },
-  "devDependencies": {
-    "@babel/cli": "^7.11.6",
-    "@babel/core": "^7.11.6",
-    "@babel/plugin-proposal-class-properties": "^7.10.4",
-    "@babel/preset-env": "^7.11.5",
-    "@babel/preset-react": "^7.10.4",
-    "@material-ui/core": "^4.11.0",
-    "@material-ui/icons": "^4.9.1",
-    "@material-ui/lab": "^4.0.0-alpha.56",
-    "@material-ui/pickers": "^3.2.10",
-    "@material-ui/styles": "^4.10.0",
-    "@storybook/addon-actions": "^5.3.21",
-    "@storybook/addon-links": "^5.3.21",
-    "@storybook/addons": "^5.3.21",
-    "@storybook/react": "^5.3.21",
-    "@storybook/storybook-deployer": "^2.8.6",
-    "babel-loader": "^8.1.0",
-    "babel-plugin-transform-dynamic-import": "^2.1.0",
-    "cross-env": "^5.2.1",
-    "flatpickr": "^4.6.6",
-    "formik": "^2.1.5",
-    "moment": "^2.29.0",
-    "prop-types": "^15.7.2",
-    "react-dropzone": "^10.2.2",
-    "react-select": "^3.1.0",
-    "rimraf": "^2.7.1",
-    "rollup-plugin-babel": "^4.4.0",
-    "rollup-plugin-commonjs": "^10.1.0",
-    "rollup-plugin-node-resolve": "^5.2.0",
-    "rollup-plugin-replace": "^2.2.0",
-    "rollup-plugin-uglify": "^6.0.4",
-    "tinymce": "^5.4.2"
-  },
-  "publishConfig": {
-    "access": "public"
-  },
-  "gitHead": "4e9bcef2c81363e46fb075d6b482117ca573dd07"
+	"name": "@settlin/formik-mui",
```

Comment:
unnecessary formatting change. please use your editor to use tabs instead of spaces, and reformat all the files